### PR TITLE
Redirects to login if not logged in(Fixes#1893)

### PR DESCRIPTION
**Pull request by SidharthBansal**

### CODE REVIEW

File: app/views/like/_like.html.erb [hunk 1]
```diff
@@ -9,9 +9,19 @@
         <% end %>
     <li class="btn btn-default btn-sm" data-html="true" rel="popover" data-content="<%= "No tags" if tagnames.nil? || tagnames.length == 0 %><% if tagnames %><% tagnames.each do |tagname| %><p style='margin-bottom:3px;'><a href='/subscribe/tag/<%= tagname %>' class='btn btn-default btn-mini'><%= tagname %></a></p><% end %><% end %><hr /><a class='btn btn-sm'><i class='fa fa-user'></i> <%= node.author.name %></a>" title="Follow by tag or author" data-placement="left"><i class="fa fa-eye"></i><span class="hidden-xs hidden-sm"> Follow</span></li>
 
-    <li rel="tooltip" title="Helpful? Like it and get updates!" class="btn btn-default btn-sm btn-like" node-id="<%= node.id %>" id="like-button-<%= node.id %>"><span id="like-star-<%= node.id %>" class="fa fa-star<% if current_user && !node.liked_by(current_user.uid) %>-o<% end %>"></span> <span id="like-count-<%= node.id %>"><%= node.likers.length %> </span></li>
+  <li rel="tooltip" title="Helpful? Like it and get updates!" class="btn btn-default btn-sm btn-like" node-id="<%= node.id %>" id="like-button-<%= node.id %>">
+    <% if !current_user %>
+      <a href = '/login' >
```
Comment (jywarren) [inline app/views/like/_like.html.erb]: Can we set a `return_to` value here? It'll get picked up by the controller after being relayed here: https://github.com/publiclab/plots2/blob/master/app/views/user_sessions/new.html.erb#L24

Comment (jywarren) [reply app/views/like/_like.html.erb]: Also, happy new year!

Comment (SidharthBansal) [reply app/views/like/_like.html.erb]: Happy New year!!!

Comment (SidharthBansal) [reply app/views/like/_like.html.erb]: Yeah that would be better for user experience to redirect back to the notes after sign up. Will change it

**Comment by SidharthBansal** [comment]
@jywarren  now it is redirecting back to the post. Please review it.
Thanks

**Comment by jywarren** [comment]
Awesome, thanks!!!

**Pull request merged**